### ISSUE TITLE
Reading: Unsupervised Domain Adaptation in Cross-corpora Abusive Language Detection

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#176]
## 0. Paper
@inproceedings{bose-etal-2021-unsupervised,
    title = "Unsupervised Domain Adaptation in Cross-corpora Abusive Language Detection",
    author = "Bose, Tulika  and
      Illina, Irina  and
      Fohr, Dominique",
    booktitle = "Proceedings of the Ninth International Workshop on Natural Language Processing for Social Media",
    month = jun,
    year = "2021",
    address = "Online",
    publisher = "Association for Computational Linguistics",
    url = "https://aclanthology.org/2021.socialnlp-1.10",
    doi = "10.18653/v1/2021.socialnlp-1.10",
    pages = "113--122",
    abstract = "The state-of-the-art abusive language detection models report great in-corpus performance, but underperform when evaluated on abusive comments that differ from the training scenario. As human annotation involves substantial time and effort, models that can adapt to newly collected comments can prove to be useful. In this paper, we investigate the effectiveness of several Unsupervised Domain Adaptation (UDA) approaches for the task of cross-corpora abusive language detection. In comparison, we adapt a variant of the BERT model, trained on large-scale abusive comments, using Masked Language Model (MLM) fine-tuning. Our evaluation shows that the UDA approaches result in sub-optimal performance, while the MLM fine-tuning does better in the cross-corpora setting. Detailed analysis reveals the limitations of the UDA approaches and emphasizes the need to build efficient adaptation methods for this task.",
}

## 2. What is amazing compared to previous works?
They analyze unsupervised domain adaptation (UDA) methods for cross-corpora abusive detection.

## 4. How did evaluate it?
- task: abuse language detection (binary classification)
- data: they used three datasets:
<img width="360" alt="スクリーンショット 2021-07-06 16 05 18" src="https://user-images.githubusercontent.com/45454055/124557016-f6e8f680-de73-11eb-94af-5f5b11d5c595.png">

- methods: they compared **five UDA methods** and one fine-tune based methods
  - PBLM: Pivot-Based Language Modeling, use a set of common features.
  - PERL-BERT: Pivot-based Encoder Representation of Language, pivot-based fine-tuning using MLM objective.
  - HATN: Hierarchical Attention Transfer Network, domain classification based adversarial training.
  - AAD-BERT: Adversarial Adaptation with Distillation with BERT.
  - HateBERT MLM fine-tune on Target: HateBERT is a pretrained BERT model using MLM objective over Reddit. 
  - HateBERT supervised fine-tune only: no-adaptation method.

Table 3 shows that all UDA methods drop their performance compared to the no-adaptation method.

<img width="719" alt="スクリーンショット 2021-07-06 16 08 35" src="https://user-images.githubusercontent.com/45454055/124557404-6d85f400-de74-11eb-8706-f57e4415a6c4.png">

## 5. Is there a discussion?
From Table 3, HateBERT with MLM fine-tuning is better than HateBERT with supervised fine-tune only.
Table 4 shows that MLM fine-tuning only target corpora is the best performance in other settings. 
<img width="397" alt="スクリーンショット 2021-07-06 16 39 03" src="https://user-images.githubusercontent.com/45454055/124561286-ade77100-de78-11eb-8905-895fc8f53893.png">
